mythril/laser/ethereum/state.py

```diff
@@ -74,11 +73,13 @@ def constraints(self):
             constraints.append(
                 ForAll(x, Implies(self[x] != 0, UGT(self.calldatasize, x)))
             )
-
+            #  10000 > calldatasize, prevents impossibly large calldatasizes
+            constraints.append(UGT(10000, self.calldatasize))
```

Comment:
I don't think this is the right place to add this, as its "technically" an untrue constraint. I'd rather put it in the concretization logic

Reply:
Also, this doesn't have to a part of this pr right?

Reply:
You're right.

I was thinking that having a massive calldata could break other things, but I guess it only really breaks concretization.